cli/cmd/uninstall.go

```diff
@@ -61,6 +61,10 @@ var uninstallCmd = &cobra.Command{
 			}
 		}
 		logging.PrintLog("Successfully uninstalled keptn", logging.InfoLevel)
+		logging.PrintLog("Note: Some namespaces and services have not been uninstalled. Please inspect this yourself using", logging.InfoLevel)
```

Comment:
This message sounds as if this was not intended.
Maybe this would be better: "Note that Keptn does not uninstall Istio, Tiller, and if installed Dynatrace."

Reply:
I wanted to be less specific about the services/namespaces that have not been uninstalled, as they might change in the future.

In that case I would go for `"Note that Keptn does not uninstall all services."`

Reply:
Being less specific makes the life of an unexperienced k8s user hard.

Reply:
Even though I have approved the PR, I also have a thought about the comments. 
I think we should tell the user what remains installed (Istio, Tiller), as we are telling the user which components get installed during the Keptn install process.
When it comes to Dynatrace, there we should check whether the namespace is available or not.

Reply:
Okay, I will rewrite the code and add some if statements.

Reply:
I've rewritten the code to list all installed namespaces and to provide a recommended action (+ a warning, where I find it appropriate).

Here is an example output:

```
Would you like to uninstall keptn from this cluster? (y/n)
y
Starting to uninstall keptn
Successfully uninstalled keptn
Note: Please review the following namespaces and perform manual deletion if necessary:
 - default
      Recommended Action: None (default namespace)
 - istio-system
      WARNING: Please consult the Istio Docs at https://istio.io/docs/setup/install/helm/#uninstall on how to remove istio completely from your cluster.
      Recommended action: kubectl delete namespace istio-system
 - kube-public
      Recommended Action: None (default namespace)
 - kube-system
      Recommended Action: Remove tiller/helm using
                          kubectl delete all -l app=helm -n kube-system
 - sockshop-dev
      WARNING: Please review this namespace in detail using 'kubectl get pods -n sockshop-dev' before deleting it
      Recommended Action: kubectl delete namespace sockshop-dev
 - sockshop-production
      WARNING: Please review this namespace in detail using 'kubectl get pods -n sockshop-production' before deleting it
      Recommended Action: kubectl delete namespace sockshop-production
 - sockshop-staging
      WARNING: Please review this namespace in detail using 'kubectl get pods -n sockshop-staging' before deleting it
      Recommended Action: kubectl delete namespace sockshop-staging

```

If a dynatrace namespace exists, it would also print the following message:
```bash

```

Reply:
Thank you! Very, very detailed.
I would have omitted the namespaces where no action is required.